loaders/translations-loader.js

```diff
@@ -0,0 +1,69 @@
+var loaderUtils = require('loader-utils');
+var handlebars = require('handlebars');
+var path = require('path');
+var hb = handlebars.create();
+
+function flatten(object) {
+  var flattened = {};
+  Object.keys(object).forEach(function(key) {
+    if (object[key] && typeof object[key] === 'object') {
+      var flatObject = flatten(object[key]);
+      Object.keys(flatObject).forEach(function(key2) {
+        flattened[[key, key2].join('.')] = flatObject[key2];
+      });
+    } else {
+      flattened[key] = object[key];
+    }
+  });
+  return flattened;
+}
+
+function compileTranslations(translations) {
+  var flattenedTranslations = flatten(translations);
+  var rows = Object.keys(flattenedTranslations).map(function(translationKey) {
+    var template = hb.precompile(flattenedTranslations[translationKey]);
+    return `${JSON.stringify(translationKey)}: (Handlebars["default"] || Handlebars).template(${template})`;
```

Comment:
Hmm, yeah didn't notice that. Tbh, i'm not 100% sure how this code works, it's based off: https://github.com/pcardune/handlebars-loader/blob/master/index.js#L287-L289